static/about.Rmd

```diff
@@ -0,0 +1,55 @@
+output:
+  html_document:
+    includes:
+      in_header: header-other.html
+      after_body: doc_afix.html
+bibliography: ../references.bib
+---
+
+```{r bibs, eval=FALSE, echo=FALSE}
+download.file("https://www.zotero.org/api/groups/418217/collections/TUEXXIV4/items/top?limit=100&format=bibtex&v=1", destfile = "references.bib")
+```
+
+This is the online home of the open source transport planning system, the Propensity to Cycle Tool (PCT). The PCT is released under the [AGP Licence](http://pct.bike/kent/licence.html): it is free to use, copy and modify, e.g. to create versions for new cities and states.
+
+The PCT was created by an academic team from [CEDAR](http://www.cedar.iph.cam.ac.uk/) (University of Cambridge), the Institute for Transport Studies ([ITS](http://www.its.leeds.ac.uk/))/[CDRC](http://cdrc.ac.uk/) (University of Leeds), the [University of Westminster](http://westminster.ac.uk/), and YouthinMind. The work was initially funded by the Department for Transport
+([DfT](https://www.gov.uk/government/organisations/department-for-transport)) to create the National Propensity to  Cycle Tool for England (PCT-England). This will be officially launched in July 2016.
+
+Further information on the project can be found at the [University of Cambridge](http://www.cedar.iph.cam.ac.uk/research/modelling/npct-tool/), the [University of Leeds](https://www.cdrc.ac.uk/research/npct/) and the [European Cycling Federation](https://ecf.com/news-and-events/news/national-propensity-cycle-tool-npct-project). The PCT was created by:
+
+- [James Woodcock](http://www.cedar.iph.cam.ac.uk/people/leads/james-woodcock/)
+- [Robin Lovelace](http://robinlovelace.net/)
+- [Rachel Aldred](http://rachelaldred.org/)
+- [Anna Goodman](http://www.lshtm.ac.uk/aboutus/people/goodman.anna)
+- [Alvaro Ullrich](http://www.cedar.iph.cam.ac.uk/people/support/alvaro-ullrich/)
+- [Ali Abbas](http://www.cedar.iph.cam.ac.uk/people/cdfs/ali-abbas/)
+- [Nikolai Berkoff](https://github.com/nikolai-b)
+
+The Propensity to Cycle Tool (PCT) was designed to assist transport planners and policy makers to prioritise investments and interventions to promote cycling. The PCT answers the question: "where is cycling currently common and where does cycling have the greatest potential to grow?"
+
+The PCT can be used at different scales.
+
+First, the PCT is a strategic planning tool. Different visons of the future are represented through the various scenarios of change, including the government's target to double cycling in a decade to the more ambitious 'Go Dutch' scenario whereby Dutch propensities to cycle (allowing for English hilliness and trip distances) are reached in England. By showing what the rate of cycling could feasibly look like in different parts of cities and regions, and illustrating the associated increase in cycle use on the road network, the PCT should inform policies that seek a wider shift towards sustainable transport more generally.
```

Comment:
change 'to the more ambitious 'Go Dutch'' to 'and the more ambitious 'Go Dutch''